modules/syslogformat/tests/test_fuzzing_syslogformat.c

```diff
@@ -0,0 +1,35 @@
+
+#include <stdint.h>
+#include <stddef.h>
+
+#include "apphook.h"
+#include "msg-format.h"
+#include "logmsg/logmsg.h"
+#include "cfg.h"
+#include "syslog-format.h"
+#include <iv.h>
+
+
+int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size)
+{
+  if (size <= 1) return 0;
+
+  GlobalConfig *cfg = cfg_new_snippet();
+  app_startup();
+  cfg_load_module(cfg, "syslogformat");
```

Comment:
How does this cfg variable works? I do not understand how it is used by syslog format. Through some side effect?

Reply:
It is not "used" by syslog format that's the trick. Rather by this `cfg_load_module` which is a proper way to load a module. This part probably could be replaced with the `syslog_format_init` function.